main/dovecot/APKBUILD

```diff
@@ -3,11 +3,11 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dovecot
-pkgver=2.2.27
+pkgver=2.2.29.1
 pkgrel=2
-_pigeonholever=0.4.16
+_pigeonholever=0.4.18
 _pluginextdataver=39
-_pkgvermajor=${pkgver%.*}
+_pkgvermajor=$(echo $pkgver | cut -d. -f1-2)
```

Comment:
I personally think it's fine to just use `cut` to deal with the varying number of dots.